x/ccv/provider/keeper/grpc_query.go

```diff
@@ -220,9 +230,9 @@ func (k Keeper) QueryThrottledConsumerPacketData(goCtx context.Context, req *typ
 // getSlashPacketData fetches a slash packet data from the store using consumerChainId and ibcSeqNum (direct access)
 // If the returned bytes do not unmarshal to SlashPacketData, the data is considered not found.
 func (k Keeper) getSlashPacketData(ctx sdk.Context, consumerChainID string, ibcSeqNum uint64) (ccvtypes.SlashPacketData, bool) {
-	store := ctx.KVStore(k.storeKey)
-	bz := store.Get(types.ThrottledPacketDataKey(consumerChainID, ibcSeqNum))
-	if len(bz) == 0 {
+	store := k.storeService.OpenKVStore(ctx)
```

Comment:
Can you revert this? Or is it needed? The storeservice is not wired correctly at this point.
```suggestion
	store := ctx.KVStore(k.storeKey)
```

Reply:
definetly. leftover from storeService move Will revert this.
Wondering if we should remove storeService from the Keeper as well as it's not used and hides such mistakes as above.